beancount_n26/__init__.py

```diff
@@ -117,6 +116,20 @@ class InvalidFormatError(Exception):
 PayeePattern = namedtuple("PayeePattern", ["regex", "account"])
 
 
+class LastLineReader:
```

Comment:
What's the main purpose behind introducing this class?

Reply:
Never mind, I added a comment [here](https://github.com/siddhantgoel/beancount-n26/pull/85#discussion_r2027491916) suggesting if we can do the same thing without defining this class.